Dockerfile

```diff
@@ -1,13 +1,21 @@
 FROM alpine
 MAINTAINER Rémi Alvergnat <toilal.dev@gmail.com>
```

Comment:
Is this maintainer field still accurate?

Reply:
No idea. @Toilal still around?